### PR TITLE
Nightly builds

### PR DESCRIPTION
**Pull request by ArangoGutierrez**
This patch fixes the playbook `nvidia-gpu-operator-ci` to be run in containerized environments and ease the way the container can be called from CI environments

### CODE REVIEW

**Comment by ArangoGutierrez** [comment]
Please first review and merge https://github.com/openshift-psap/ci-artifacts/pull/16 before this PR

**Comment by ArangoGutierrez** [comment]
@kpouget @sjug Let's continue the conversation from #16 here

File: roles/openshift_node/tasks/create_machine.yml [hunk 1]
```diff
@@ -3,13 +3,12 @@
 
 - name: Create OpenShift object
   command: >
-    oc create -f {{ gpu_machineset_deff }}
-    --kubeconfig={{ kubeconfig }}
+    bash -c "oc create -f {{ gpu_machineset_deff }} --kubeconfig '{{ kubeconfig }}'"
```
Comment (kpouget) [inline roles/openshift_node/tasks/create_machine.yml:6]: @ArangoGutierrez could you quote this one as well, please?

Comment (ArangoGutierrez) [reply roles/openshift_node/tasks/create_machine.yml]: done

**Comment by kpouget** [comment]
the code of the patch looks good to me, but tomorrow I'll try to understand more what's it's doing&changing

**Review by kpouget — COMMENTED** [review]
(ignore)

File: build/root/usr/local/bin/run [hunk 2]
```diff
@@ -5,9 +5,7 @@
 #
 # SOURCE and HOME DIRECTORY: /opt/ci-artifacts/src
 
-set -o nounset
 set -o errexit
-set -o pipefail
```
Comment (kpouget) [inline build/root/usr/local/bin/run:10]: why did you remove these flags? 

> Use `set -o pipefail` in scripts to catch mysqldump fails in e.g. mysqldump |gzip. The exit status of the last command that threw a non-zero exit code is returned.

> Use `set -o nounset` (a.k.a. `set -u`) to exit when your script tries to use undeclared variables.

except if there is a very good reason behind it, better keeping them I think

**Review by kpouget — COMMENTED** [review]
seems good to me, except that I don't understand why the bash flags were removed

File: build/root/usr/local/bin/run (hunk 2 above)
Comment (ArangoGutierrez) [reply build/root/usr/local/bin/run]: https://github.com/openshift-psap/ci-artifacts/pull/17/commits/1dd4636644489e884e8785d21aea01cf3d9f0470

**Comment by kpouget** [comment]
/lgtm

thanks Eduardo!
